integration-tests/ssh/src/test/java/org/apache/camel/quarkus/component/ssh/it/SshTestResource.java

```diff
@@ -45,33 +54,77 @@ public Map<String, String> start() {
             container = new GenericContainer(SSH_IMAGE)
```

Comment:
If we're going to use the embedded SSHD server, then we should probably replace usage of the container with the embedded server for consistency.

Could done as a future follow up.

Reply:
Reported as https://github.com/apache/camel-quarkus/issues/6917